sopt30th_1st HW/sopt30th_1st HW/LoginViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  LoginViewController.swift
+//  sopt30th_1st HW
+//
+//  Created by 최영린 on 2022/04/06.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+
+    @IBOutlet weak var idTextField: UITextField!
+    @IBOutlet weak var pwTextField: UITextField!
+    @IBOutlet weak var loginButton: UIButton! {
+        didSet{
+            loginButton.isEnabled = false
+        }
+    }
+    @IBOutlet weak var passwordButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setBackButton()
+        setLoginButton()
+    }
+    
+    @IBAction func passwordButton(_ sender: Any) {
+        if(pwTextField.isSecureTextEntry){
+            pwTextField.isSecureTextEntry = false
+            passwordButton.setImage(UIImage(named: "PasswordHidden"), for: .normal)
+            }
+        else{
+            pwTextField.isSecureTextEntry = true
+            passwordButton.setImage(UIImage(named: "PasswordShown"), for: .normal)
+        }
+    }
```

Comment:
이 부분에서 쓸 수 있는 삼항연산자라는 것을 소개해드리고 싶네요~!

자세히 찾아보시면 정말 편하게 사용할 수 있고 좋다고 생각하실텐데,, 혹시나 해서 간단히 (대충...?) 설명드리자면
A ? B : C
A가 참이면 B를 하고
A가 거짓이면 C를 하라는 겁니다..!

```suggestion
    @IBAction func passwordButton(_ sender: Any) {
    pwTextField.isSecureTextEntry ? passwordButton.setImage(UIImage(named: "PasswordHidden"), for: .normal) : passwordButton.setImage(UIImage(named: "PasswordShown"), for: .normal)
    pwTextField.isSecureTextEntry = !pwTextField.isSecureTextEntry
    }
```


혹은~


```suggestion
    @IBAction func passwordButton(_ sender: Any) {
    passwordButton.setImage(pwTextField.isSecureTextEntry ? UIImage(named: "PasswordHidden") : UIImage(named: "PasswordShown"), for: .normal)
    pwTextField.isSecureTextEntry = !pwTextField.isSecureTextEntry
    }
```



이렇게 될 수도 있겠네용~!

Reply:
이렇게 자세한 코드리뷰 감사합니다!!!! 정말 많이 배워가요!